package.json

```
@@ -87,6 +87,12 @@
         "title": "Edit Connection",
         "category": "Confluent: Connections"
       },
+      {
+        "command": "confluent.connections.direct.export",
+        "icon": "$(desktop-download)",
+        "title": "Save connection configuration as json file",
```

```suggestion
        "title": "Save Connection as JSON",
```
Thoughts on a shorter title?